judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/action/getActionInputRangeBehaviour.etl

```diff
@@ -167,4 +176,72 @@ rule CreateCardinalityForGetRangeActionInputEntityTypeOutput
         t.lower = 0;
         t.upper = -1;
 }
+*/
+
+// rules for range input type for input paramter (old solution)
+
+@lazy
+@greedy
+rule CreateCardinalityForGetRangeActionInputTransferTypeInputParameterOwner
+    transform s: JSL!TransferActionDeclaration
+    to t : JUDOPSM!Cardinality {
+        t.setId("(jsl/" + s.getId() + ")/CreateCardinalityForGetRangeActionInputTransferTypeInputParameterOwner");
+        t.lower = 0;
+        t.upper = 1;
+}
+
+@lazy
+@greedy
+rule CreateCardinalityForGetRangeActionInputTransferTypeInputParameterQueryCustomizer
+    transform s: JSL!TransferActionDeclaration
+    to t : JUDOPSM!Cardinality {
+        t.setId("(jsl/" + s.getId() + ")/CreateCardinalityForGetRangeActionInputTransferTypeInputParameterQueryCustomizer");
+        t.lower = 0;
+        t.upper = 1;
+}
 
+@lazy
+@greedy
+rule CreateTransferObjectRelationForGetRangeActionInputTransferTypeInputParameterOwner
+    transform s: JSL!TransferActionDeclaration
+    to t : JUDOPSM!TransferObjectRelation {
+        t.setId("(jsl/" + s.getId() + ")/GetRangeActionInputTransferTypeInputParameterOwner");
```

Comment:
> /GetRangeActionInputTransferTypeInputParameterOwner

/CreateTransferObjectRelationForGetRangeActionInputTransferTypeInputParameterOwner ?

Reply:
In my opinion, the ID will be long, but let it be

Reply:
[59ae12b](https://github.com/BlackBeltTechnology/judo-tatami-jsl/pull/314/commits/59ae12b4f85e82ed18322fc8fad4e90bb6bfd026)